thirdeye/thirdeye-pinot/pom.xml

```diff
@@ -17,6 +17,12 @@
   </properties>
 
   <dependencies>
+    <!-- https://mvnrepository.com/artifact/nz.ac.waikato.cms.weka/weka-dev -->
```

Comment:
I believe there is not need to include this here. We can use weka for now in our internal lib, and there is no concerns at all. Will wrote if we really want to open-source alpha-beta-logistic classifier.